### PR TITLE
Correct Nsquared unit

### PR DESCRIPTION
**Pull request by rcaneill**
The unit of Nsuared should be 1/s^2

### CODE REVIEW

File: gsw/stability.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def Nsquared(SA, CT, p, lat=None, axis=0):
     Returns
     -------
     N2 : array
-        Buoyancy frequency-squared at pressure midpoints, 1/s.
+        Buoyancy frequency-squared at pressure midpoints, 1/s^2.
```
Comment (ocefpaf) [inline gsw/stability.py:50]: I wonder if we should add the same text from the docs in http://www.teos-10.org/pubs/gsw/html/gsw_Nsquared.html

```
The units of N2 are radians2 s-2 however in may textbooks this is
 abreviated to s-2 as radians does not have a unit.  To convert the
 frequency to hertz, cycles sec-1, divide the frequency by 2π, ie N/(2π).
```

Comment (rcaneill) [reply gsw/stability.py:50]: Seems a good idea, does the license authorizes it? If so I'll be happy to commit this

Comment (ocefpaf) [reply gsw/stability.py:50]: Our current docstrings are extracted from the matlab ones, used for that page. I'm not sure why this one mismatches and it is hard to follow the matlab code b/c it is not versioned.

TL;DR I believe it is OK. What do you think @efiring?

**Comment by rcaneill** [comment]
Any idea why the windows test does not pass?

File: gsw/stability.py [hunk 2]
```diff
@@ -47,9 +47,12 @@ def Nsquared(SA, CT, p, lat=None, axis=0):
     Returns
     -------
     N2 : array
-        Buoyancy frequency-squared at pressure midpoints, 1/s.
+        Buoyancy frequency-squared at pressure midpoints, rad^2/s^2.
         The shape along the pressure axis dimension is one
         less than that of the inputs.
+        The units of N2 are radians2 s-2 however in may textbooks this is
```
Comment (efiring) [inline gsw/stability.py:53]: I would leave out this block. Surely the audience for gsw can be expected to know the difference between radians and cycles. Specification of s^{-2} as the units for N^2 is not an abbreviation and is not restricted to textbooks; it is standard.  Where cycles are desired as the unit of phase, it is explicit, e.g., Hz, or cycles per hour.   But if you really want to be more explicit here, I would change the "rad^2/s^2" in the top changeset line to "s^{-2}", and then say, "(Frequency N is in radians per second.)", and leave it at that.

**Review by efiring — CHANGES_REQUESTED** [review]

**Comment by ocefpaf** [comment]
> The windows test failure is at the installation stage. Pip is installing a bunch of dependencies, and installation of pyparsing fails. I think all of these dependencies should be obtainable from conda-forge. In any case, it looks like some tweaking of the windows test setup will be needed.

The problem is with latest `pip`. See https://github.com/conda-forge/pip-feedstock/issues/94

PR https://github.com/conda-forge/admin-requests/pull/439 should fix.

**Comment by rcaneill** [comment]
Damn it, the build and deploy action did not work... It says broken link but the links work for me... Maybe re-trigger this test manually?

**Comment by ocefpaf** [comment]
> Damn it, the build and deploy action did not work... It says broken link but the links work for me... Maybe re-trigger this test manually?

Don't worry. That is pretty minor, it is only the linkcheck telling us that the TEOS-10 site moved. I'll fix it in a bit. We can merge this without that check if @efiring is ok with the changes.

**Review by efiring — APPROVED** [review]
